plugins/aladino/actions/review.go

```diff
@@ -21,17 +24,35 @@ func Review() *aladino.BuiltInAction {
 
 func reviewCode(e aladino.Env, args []aladino.Value) error {
 	t := e.GetTarget().(*target.PullRequestTarget)
+	reviewer := e.GetGithubActionActor()
+
+	log.Printf("%+v", reviewer)
 
 	reviewEvent, err := parseReviewEvent(args[0].(*aladino.StringValue).Val)
 	if err != nil {
 		return err
 	}
 
 	reviewBody, err := checkReviewBody(reviewEvent, args[1].(*aladino.StringValue).Val)
+
+	reviews, err := t.GetReviews()
 	if err != nil {
 		return err
 	}
 
+	if codehost.HasReview(reviews, reviewer) {
+		lastReview := codehost.LastReview(reviews, reviewer)
+
+		if lastReview.State == "APPROVED" {
+			return nil
+		}
+
+		log.Printf("%+v", t.PullRequest.GetUpdatedAt())
```

Comment:
If we are keeping this log we need to make clear what it is. The way it is we will see a date but is not clear that is that is about.

```suggestion
		log.Printf("Pull request updated at: %+v", t.PullRequest.GetUpdatedAt())
```